specification/dataprotection/resource-manager/Microsoft.DataProtection/preview/2021-02-01-preview/examples/BackupInstanceOperations/FindRestorableTimeRanges.json

```diff
@@ -0,0 +1,34 @@
+{
+    "parameters": {
+      "subscriptionId": "04cf684a-d41f-4550-9f70-7708a3a2283b",
+      "resourceGroupName": "Blob-Backup",
+      "vaultName": "ZBlobBackupVaultBVTD3",
+      "backupInstances": "zblobbackuptestsa58",
+      "api-version": "2020-01-01-alpha",
+      "parameters": { 
+          "objectType": "AzureBackupFindRestorableTimeRangesRequest",
+          "sourceDataStoreType": "OperationalStore",
+          "startTime": "2020-10-17T23:28:17.6829685Z",
+          "endTime": "2021-02-24T00:35:17.6829685Z"
+      }
+    },
+    "responses": {
+        "200": {
+          "body": {
+            "id": "zblobbackuptestsa58",
```

Comment:
>"id [](start = 12, length = 3)

Is this ARM resource ID? Then it should be a full resource ID. [https://armwiki.azurewebsites.net/introduction/concepts/resourceids.html](https://armwiki.azurewebsites.net/introduction/concepts/resourceids.html)

Reply:
RP should always return ID as a fully qualified resource ID, which is the platform features (e.g. policy) expect. Since this is a POST action, it may be okay but please consider returning a full id in future.